src/IndividualReview.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useLocation, useParams } from "react-router";
+import { Link } from "react-router-dom";
+
+const IndividualReview = () => {
+  let { reviewId } = useParams();
+  const [review, setReview] = useState("");
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://fame-boardgame-review-website.onrender.com/api/reviews/${reviewId}`
+      )
+      .then(({ data }) => {
+        setReview(data.review);
+      });
+  }, []);
+
+  return (
+    <div className="individualReview">
+      <div>
+        <Link to={"/"}>
+          <button>Back to homepage</button>
+        </Link>
+      </div>
+      <p className="owner">{review.owner}</p>
+      <p className="date">Date: {review.created_at}</p>
```

Comment:
You may want to create a util function for displaying the data in a more user-friendly way here. Or use some kind of string method?